src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CallbackHandlerFactory.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.exceptions.CallbackException;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.CallbackHandler;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.CallbackParams;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.GenerateOrderCallbackHandler;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.MoreTimeRequestedCallbackHandler;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.GENERATE_ORDER;
+import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.MORE_TIME_REQUESTED_PAPER;
+
+@Service
+public class CallbackHandlerFactory {
+
+    private Map<String, CallbackHandler> handlers = new HashMap<>();
+
+    @Autowired
+    public CallbackHandlerFactory(
+        MoreTimeRequestedCallbackHandler moreTimeRequestedCallbackHandler,
+        Optional<GenerateOrderCallbackHandler> generateOrderCallbackHandler) {
+        handlers.put(MORE_TIME_REQUESTED_PAPER.getValue(), moreTimeRequestedCallbackHandler);
+        generateOrderCallbackHandler.ifPresent(h -> handlers.put(GENERATE_ORDER.getValue(), h));
```

Comment:
Please fix the formatting;
```
MethodOrConstructor(
    Type1 param1,
    Type2 param2,
     ....
) {
     body
}
```
& GenerateOrderCallbackHandler
& MoreTimeRequestedCallbackHandler
& DocAssemblyTemplateBodyMapper (maybe)

Reply:
all of this is gone, sorry, not part of this pr :(